recipes/qiskit-aer/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "qiskit-aer" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Qiskit/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 191b22dfcfbe6ed60e7638c9001b3ec685faf21daa7219a9bbd811fe6afd696e
+  patches:
+    # This patch is used to prevent scikit-build from trying to pip install cmake.
+    - 0001-Drop-setup_requires.patch
+    # This patch is used to exclude extraneous files from the package.
+    - 0002-Specify-package_data-explicitly.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
+  skip: True  # [unix and py>310]
+  number: 0
+  script_env:
+    - DISABLE_DEPENDENCY_INSTALL=ON
+    - DISABLE_CONAN=ON
+    - SKBUILD_CONFIGURE_OPTIONS="-GNinja"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake !=3.17,>=3.8
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+    - ninja
+  host:
+    - cmake !=3.17,>=3.8
+    - libblas
+    - liblapack
+    - muparserx =4.0.8
+    - nlohmann_json =3.1.2
+    - numpy >=1.16.3
+    - pip
+    - python
+    - pybind11 >=2.6
+    - scikit-build >=0.11.0
+    - setuptools >=40.1.0
+    - spdlog =1.5.0
+  run:
+    - {{ pin_compatible('numpy') }}
+    - muparserx =4.0.8
```

Comment:
```suggestion
```
The run_export from the muparserx package should ensure that the correct version is available at runtime. If it doesn't, then your run_export is wrong.